interpreter/interpreter_test.go

```diff
@@ -90,8 +90,11 @@ func TestProcessDeclarations(t *testing.T) {
 			DirectorType: &ast.Ident{Value: "client"},
 			Properties: []ast.Expression{
 				&ast.DirectorProperty{
-					Key:   &ast.Ident{Value: "quorum"},
-					Value: &ast.String{Value: "20%"},
+					Key: &ast.Ident{Value: "quorum"},
```

Comment:
need `go fmt` probably

Reply:
Since the `Value` literal is now on multiple lines the alignment of `Key` with it was actually removed by `go fmt`.